pastepwn/analyzers/googleapikeyanalyzer.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+from .regexanalyzer import RegexAnalyzer
+
+
+class GoogleApiKeyAnalyzer(RegexAnalyzer):
+
+    def __init__(self, actions):
+        # https://cloud.google.com/docs/authentication/api-keys
+        regex = r"^AIza[0-9A-Za-z_-]{35}$"
```

Comment:
Keys can also be in the middle of the paste body, so it would be better to replace `^` and `$` with `\b`.
Also a new test case to check if the google api key will be matched in the middle of a sentence would be appreciated a lot.